### PR TITLE
add setTcpdfLink method to allow enabling or disabling TCPDF meta link

### PR DESCRIPTION
**Pull request by it-can**
add setTcpdfLink method to allow enabling or disabling TCPDF meta link

### CODE REVIEW

**Comment by it-can** [comment]
> Why did you close this one ?

oops, reopend it

File: tcpdf.php [hunk 1]
```diff
@@ -1973,7 +1973,6 @@ public function __construct($orientation='P', $unit='mm', $format='A4', $unicode
 		$this->setCompression();
 		// set default PDF version number
 		$this->setPDFVersion();
-		$this->tcpdflink = true;
```
Comment (williamdes) [inline tcpdf.php:1976]: maybe this change is not needed

File: tcpdf.php [hunk 2]
```diff
@@ -2454,6 +2453,15 @@ public function setLastH($h) {
 		$this->lasth = $h;
 	}
 
+	/**
+	 * Allow to disable/enable TCPDF meta link
+	 * @param bool $tcpdflink
+	 * @public
```
Comment (williamdes) [inline tcpdf.php:2459]:
```suggestion
```

Comment (williamdes) [inline tcpdf.php]:
```suggestion
	 * @return void
```

**Review by williamdes — APPROVED** [review]
This looks okay except the removed line

**Review by williamdes — APPROVED** [review]

**Comment by nicolaasuni** [comment]
Please try to resolve the conflicts.

**Comment by it-can** [comment]
@nicolaasuni the code was messed up, created a new PR #713

**Pull request closed without merging**